<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WAudioPlayer</class>
 <widget class="QWidget" name="WAudioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>48</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>48</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>48</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="previousButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/prev.svg</normaloff>:/icons/prev.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stopButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/stop.svg</normaloff>:/icons/stop.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="playButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/play.svg</normaloff>:/icons/play.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/next.svg</normaloff>:/icons/next.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>previousButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>nextButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
